output.csv

```diff
@@ -0,0 +1,3 @@
+type,fruit
```

Comment:
move these files to resources

Reply:
not fixed

Reply:
> not fixed

![image](https://github.com/mate-academy/jv-fruit-shop/assets/54332495/9d7e29e3-1bd0-4a53-8484-a1625ec2315d)

Reply:
Not fixed. You have two output.csv files: one in the resources folder, one in the root folder
Delete **output.csv** from the root folder